templates/partials/head.hbs

```diff
@@ -1,6 +1,17 @@
-<title>{{#if pageTitle }}{{pageTitle}}{{else}}{{site.title}}{{/if}}</title>
 <meta charset="UTF-8">
+<title>{{#if pageTitle }}{{pageTitle}}{{else}}{{site.title}}{{/if}}</title>
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
 <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
 <meta name="description" content="{{description}}">
-<link rel="stylesheet" href="/styles/style.css">
\ No newline at end of file
+<meta property="og:title" content="{{#if ogTitle }}{{ogTitle}}{{else}}{{title}}{{/if}}">
+<meta property="og:site_name" content="{{site.title}}">
+<meta property="og:type" content="{{ogType}}">
+<meta property="og:url" content="{{site.url}}/{{#if path}}{{path}}/{{/if}}">
+<meta property="og:image" content="{{site.url}}/images/{{ogImage}}">
+<meta property="og:image:alt" content="{{ogImageAlt}}">
+
+<meta name="twitter:card" content="{{twitterCard}}" />
+<meta name="twitter:site" content="{{twitterSite}}" />
+
```

Comment:
why not having `twitter:creator` too pointing to the company's Twitter account?

Reply:
Don't think that property is appropriate in this case. According to [Twitter's docs](https://developer.twitter.com/en/docs/tweets/optimize-with-cards/guides/getting-started):

> *Creator Attribution:* Indicates the _individual user_ that created the content within the card. This applies to the Summary with Large Image card.

Once we have things like blog posts, case studies, etc. we can add that with the Twitter handle of whoever authored those things. But all our current pages are generic ones so I omitted it.

Reply:
well but you still can say that the content is related to the twitter account of buildit... I'm not fussed about it sincerely, I'm just wondering what will change when tweeting with this or without it. My assumption is that the author is going to be linked when sharing, by going with this assumption, it can only be better, and the interpretation can be a bit loose 🤷‍♂️

Reply:
I'd like to leave it as is for now and test it once it goes live on the beta URL. I agree we want some kind of link from the card to our Twitter account. However, I'm assuming that the `twitter:site` attribution by itself will achieve this. If not, I'll add the `creator` one (or do whatever to get the desired effect). I just don't want to add duplicated info unnecessarily.